### PR TITLE
Update npgsql to 4.0.17 from 4.0.10

### PR DESCRIPTION
**Pull request by vleslief-ms**
The next release of PowerBI will ship with npgsql 4.0.17 instead of 4.0.10. PostgreSQL connector document should be updated to reflect that.

### CODE REVIEW

**Comment by Jak-MS** [comment]
@DougKlopfenstein 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: powerquery-docs/connectors/postgresql.md [hunk 1]
```diff
@@ -23,11 +23,11 @@ ms.author: dougklo
```
Comment (KazimMir) [inline powerquery-docs/connectors/postgresql.md:23]: Should you add yourself in the author?

**Review by KazimMir — APPROVED** [review]
LGTM. Minor question

**Comment by Jak-MS** [comment]
@KazimMir @DougKlopfenstein - this is still waiting for release, correct?  can you sign off when it's ready for merge? thanks.

File: powerquery-docs/connectors/postgresql.md (hunk 1 above)
Comment (DougKlopfenstein) [reply powerquery-docs/connectors/postgresql.md:23]: I'm already in the metadata as the author and ms.author.

**Comment by vleslief-ms** [comment]
Due to some issues with 4.0.16, PBI Desktop had to be updated to npgsql 4.0.17 instead. Will update PR to reflect that.

**Comment by vleslief-ms** [comment]
Document has been updated to reflect 4.0.17 version usage. Due to the releases page on npgsql not including a 4.0.17 MSI, I rewrote a little bit of the later paragraph to be more generic.

**Review by DougKlopfenstein — APPROVED** [review]

**Pull request merged**